ejb3/src/main/java/org/jboss/as/ejb3/remote/EJBRemoteConnectorService.java

```diff
@@ -68,7 +68,7 @@
 public class EJBRemoteConnectorService implements Service<EJBRemoteConnectorService> {
 
     // TODO: Should this be exposed via the management APIs?
-    private static final String EJB_CHANNEL_NAME = "jboss.ejb";
+    private static final String EJB_CHANNEL_NAME = "ejb";
```

Comment:
I don't think this change is needed. The EJB service type was fixed in EJBClientChannel:

https://github.com/jbossas/jboss-ejb-client/pull/156